### DISCUSSION TITLE
unable to play song from localhost using setSourceUrl but able to using setSourceBytes

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [bluefireteam/audioplayers#1561] [Q&A]
I am unable to play songs from the HLS link which is hosted locally using the setSourceUrl function but able to play songs, but when I tested manually by playing the song using setSourceBytes [sending HTTP request and converting the response to bytes and passing]. but it will be a problem to play song manually so how can i play song using setSourceUrl

**Comment by Gustl22** [comment]
Can you post an example url and what platform you are using? What version of AP do you use?
